src/lib/components/forms/select-box/terra-select-box.component.ts

```diff
@@ -335,6 +305,14 @@ export class TerraSelectBoxComponent implements OnInit, OnChanges
         this.toggleOpen = false;
     }
 
+    private sortListBoxValues():void
+    {
+        if(!this.disableSorting && this.inputListBoxValues && this.inputListBoxValues.length > 0)
```

Comment:
Die beiden letzten Fälle sollten in der `sortArray` Methode des `SelectBoxSortHelper` abgefangen werden.